inc/admin.php

```diff
@@ -272,6 +272,12 @@ function settings_screen( $post_id, $strings, $args ) {
 	clean_orphaned_posts( $post_id, $args['post_type'] );
 	?>
 <div class="wrap">
+	<div class="notice notice-error adstxt-notice existing-adstxt" style="display: none;">
+		<p><strong><?php echo esc_html( $strings['existing'] ); ?></strong></p>
+		<p><?php echo esc_html( $strings['precedence'] ); ?></p>
+
+		<p><?php echo esc_html_e( 'Removed the existing file but are still seeing this warning?', 'ads-txt' ); ?> <a class="ads-txt-rerun-check" href="#"><?php echo esc_html_e( 'Re-run the check now', 'ads-txt' ); ?></a> <span class="spinner" style="float:none;margin:-2px 5px 0"></span></p>
+	</div>
```

Comment:
Follow up: it's the presence of this that is causing the tests to fail. I pushed a fix in 5d9f27dfab8599b8d708d747a1e7647b71fb6794